addons/im_livechat/models/discuss_channel.py

```diff
@@ -25,6 +25,15 @@ class DiscussChannel(models.Model):
     chatbot_current_step_id = fields.Many2one('chatbot.script.step', string='Chatbot Current Step')
     chatbot_message_ids = fields.One2many('chatbot.message', 'discuss_channel_id', string='Chatbot Messages')
     country_id = fields.Many2one('res.country', string="Country", help="Country of the visitor of the channel")
+    conversation_outcome = fields.Selection(
+        selection=[
+            ("success", "Success"),
+            ("never_answered", "Never Answered"),
+            ("bot_could_not_forward", "Bot Could Not Forward"),
+            ("escalated", "Escalated"),
+        ],
+        string="Conversation Outcome",
+    )
```

Comment:
This should be named "livechat_outcome" for clarity this is a state specific to livechat (& chatbot).

Reply:
I would rename selection items as follow:

```
success
no_answer
bot_fail_to_forward
bot_forwarded
```

For:
- more consistent wording ("forwarded"),
- easier to read ("no_answer", "fail_to_forward"),
- prefix for chatbot-specific states.